name-badges/makebadges.py

```diff
@@ -0,0 +1,40 @@
+#!/bin/pyhton3
+
+import sys
+import re
+import lxml.etree as etree
+import subprocess
+
+namesfile = sys.argv[1]
```

Comment:
Minor: I'd encourage using something like [`argparse`](https://docs.python.org/3/library/argparse.html) to aid usability.

Reply:
[Click](https://click.palletsprojects.com/en/7.x/) is also nice, but is an additional dependency.